File: protected/controllers/MessageController.vb
Imports GeoPortal.GeoPortal.Models.BusinessLogic
Imports GeoPortal.GeoPortal.Models.Data

Namespace GeoPortal
    Public Class MessageController
        Inherits System.Web.Mvc.Controller

        '
        ' GET: /Message

        Function submitAbug(ByVal name As String, ByVal email As String, ByVal sendDate As String, ByVal message As String, ByVal activity As String) As JsonResult

            Dim msg As New sendEmail()
            Dim result As New jsonMsg

            Dim msgbody As New StringBuilder()
            msgbody.AppendLine("From: " & name & "</br>")
            msgbody.AppendLine("Email: " & email & "</br>")
            msgbody.AppendLine("Report:" & "</br>")
            msgbody.AppendLine(message & "</br>")
            msgbody.AppendLine("Activity:" & "</br>")
            msgbody.AppendLine(activity & "</br>")


            If (msg.SendEmail(email, "WISERD@glam.ac.uk", "", "Bug Report", msgbody.ToString())) Then
                result.success = True
                result.message = "Thankyou " + name + ",your bug report has been sucessfully submited."

            Else
                result.success = False
                result.message = "Error Sending Bug report, please try again!"

            End If


            Return Json(result)
        End Function

    End Class
End Namespace

File: protected/controllers/MessageController.php

<?

class MessageController extends Controller {

        Function actionsubmitAbug() {

            $name = 'null';
            $email = 'null';
            $message = 'null';
            $activity = 'null';

            if(isset($_POST['name'])) {
                $name = $_POST['name'];
            }
            if(isset($_POST['email'])) {
                $email = $_POST['email'];
            }
            if(isset($_POST['message'])) {
                $message = $_POST['message'];
            }
            if(isset($_POST['activity'])) {
                $activity = $_POST['activity'];
            }
//            $msg = New sendEmail();
            $result = New jsonMsg;

            $msgbody = "";
            $msgbody .= ("From: " . $name . "<br><br>");
            $msgbody .= ("Email: " . $email . "<br><br>");
            $msgbody .= ("Report:" . "<br>");
            $msgbody .= ($message . "<br><br>");
            $msgbody .= ("Activity:" . "<br>");
            $msgbody .= ($activity . "<br><br>");

            $sent = sendEmail::SendBugEmail($msgbody);

            If ($sent) {
                $result->success = True;
                $result->message = "Thank you " . $name . ", your bug report has been successfully submitted.";

            } Else {
                $result->success = False;
                $result->message = "Error Sending Bug report, please try again!";

            }


            echo json_encode($result);
        }

}
